### ISSUE TITLE
SVM

### ISSUE THREAD

**Issue by chalidazia** [chalidazia/Tugas-Akhir#1]
import numpy as np
import pandas as pd
from sklearn.model_selection import KFold
from sklearn.svm import SVC
from sklearn.metrics import accuracy_score, confusion_matrix

data = pd.read_csv("tes.csv")
arr = np.array(data)

X = arr[:,:5800]
y = arr[:,-1]

kf = KFold(n_splits=5)
kf.get_n_splits(X)

# print(kf)

KFold(n_splits=5, random_state=None, shuffle=False)

for train_index, test_index in kf.split(X):
    # print("TRAIN:", train_index, "TEST:", test_index)
    X_train, X_test = X[train_index], X[test_index]
    y_train, y_test = y[train_index], y[test_index]

# X_train, X_test, y_train, y_test = train_test_split(X, y, test_size = 0.20, random_state = 0)

svclassifier = SVC(kernel='rbf', gamma='auto')
svclassifier.fit(X_train, y_train)

# TODO: jangan lupa sava variabel svclassifier sebagai model untuk menghindari proses training berkali-kali
# pada saat menjalankan file ini

y_pred = svclassifier.predict(X_test)

print(accuracy_score(y_test, y_pred))
print(confusion_matrix(y_test, y_pred))
# print(classification_report(y_test, y_pred))
